tests/IoTeX/AddressTests.cpp

```diff
@@ -45,8 +45,8 @@ TEST(IoTeXAddress, FromPrivateKey) {
     EXPECT_THROW({
         try
         {
-            const auto publicKey = PublicKey(privateKey.getPublicKey(TWPublicKeyTypeSECP256k1));
-            const auto address = Address(publicKey);
+            const auto publicKey_ = PublicKey(privateKey.getPublicKey(TWPublicKeyTypeSECP256k1));
```

Comment:
when to use `name_`? we all agree `_name` and `gName`

Reply:
> when to use `name_`? we all agree `_name` and `gName`

Since `_field` is for field member, we didn't agree on one yet on a scoped variable, choose `scoped_`, can change it if required

Reply:
since it's scoped, maybe just `scoped`? no more underscore

Reply:
`_name` and `name_` is quite confusing, error prone, hard to enforce, no way to automatically enforce.
For my simple mental model: `_name` for anything where `name` cannot be used for some reason